src/msw/window.cpp

```diff
@@ -539,6 +539,19 @@ bool wxWindowMSW::CreateUsingMSWClass(const wxChar* classname,
     msflags &= ~WS_BORDER;
 #endif // wxUniversal
 
+    // Enable double buffering by default for all our own, i.e. not the ones
+    // using native controls, classes.
+    //
+    // Note that this function is not used for top-level windows, so we don't
+    // set this style for them, and also that setting it for children of
+    // windows that already have WS_EX_COMPOSITED set doesn't seem to have any
+    // bad effect as the style is just ignored in this case, so we don't bother
+    // checking it it's already set for the parent, even though we could.
+    if ( !classname )
+    {
+        exstyle |= WS_EX_COMPOSITED;
+    }
+
```

Comment:
Tried this PR with the `wxGrid` control and it doesn't seem to work
because this style `WS_EX_COMPOSITED` is not applied for it (or its children)  !
i.e. `classname` is != NULL and has this value "`wxWindowNR`", what does that mean ?

Reply:
No, it's a false alarm sorry.
I just recompiled everything from the beginning and I confirm that it works perfectly well
sorry again for the noise!